blues/example.py

```diff
@@ -11,12 +11,11 @@
 """
 
 from __future__ import print_function
-from blues import ncmc
-from blues import utils
+from blues.moves import RandomLigandRotationMove, MoveEngine
```

Comment:
If we add the imports in the`__init__` do we have to import it like this still? Couldn't it just be `from moves import RandomRotationMove`?